cde-pentaho5/resource/resources/custom/amd-components/Dashboard/DashboardComponent.js

```diff
@@ -78,9 +78,18 @@ define([
             if(myself.isParameterPublic(otherParam)) {
               var eventName = myParam + ":fireChange";
               var fun = function(evt) {
-                reqDash.fireChange(otherParam, evt.value);
+                if(reqDash.getParameterValue(otherParam) !== evt.value) {
```

Comment:
I think we should add a test to validate that **type conversion** doesn't happen.